js/scripts.js

```diff
@@ -1,23 +1,242 @@
+                <h2 class="modal__title">${name}</h2>
+                <button class="modal__button">
+                  <img
+                    id="modal__button--close"
+                    src="./images/icons/dark/x-icon-dark.svg"
+                    alt="PopUp window close button"
+                  />
+                </button>
+              </header>
+              <div class="modal__content flex-column">
+                <div class="modal__profile flex-row align-center">
+                  <h3 class="uppercase color-info">${profile}</h3>
+                  <ul class="modal__list flex-row">
+                    ${categories
+                      .map((item) => `<li class="modal__item">${item}</li>`)
+                      .join('')}
+                  </ul>
+                </div>
+                <img
+                  src="${image}"
+                  alt="Work 1"
+                  class="modal__image"
+                />
+               <div class="modal__system grid grid-uneven-two">
+                <p class="modal__description">
+                  ${description}
+                </p>
+                <div class="modal__system flex-column ">
+                  <ul class="modal__tags flex-row align-center">
+                    ${technologies
+                      .map((item) => `<li class="modal__tag">${item}</li>`)
+                      .join('')}
+                  </ul>
+                  <hr />
+                  <div class="modal__actions flex-row">
+                    <a
+                      id="0"
+                      href="${live}"
+                      target="_blank"
+                      class="button button__primary modal__button flex-row align-center justify-center"
+                      >See live 
+                      <img
+                        src="./images/icons/dark/live-dark.svg"
+                        alt="Live icon dark version"
+                      /></a
+                    >
+                    <a
+                      id="0"
+                      href="${source}"
+                      target="_blank"
+                      class="button button__primary modal__button flex-row align-center justify-center"
+                      >See source
+                        <img
+                        src="./images/icons/dark/github-dark.svg"
+                        alt="Github icon dark version"
+                      />
+                      </a
+                    >
+                  </div>
+                </div>
+               </div>
+              </div>
+            </article>
+         `;
+
+    bodyElement.style.overflow = 'hidden';
+    bodyElement.appendChild(modalItem);
+
+    const modalCloseButton = document.querySelector('#modal__button--close');
+    modalCloseButton.addEventListener('click', () => {
+      bodyElement.removeChild(modalItem);
+      bodyElement.style.overflow = 'auto';
+    });
+  }
```

Comment:
- [ ] Please add a scroll-down functionality to the modal on mobile view as you did on desktop view. AS you can see in the image below, I cannot scroll done to click or tap on the buttons. 

<img width="188" alt="image" src="https://github.com/Rysth/Portfolio-Project/assets/84156714/6bb70845-2d01-4d40-87e0-78a766ebfb02">

Reply:
Thanks, @bobb-Rob!

### 🏅 Fixeds Implemented

List of things that were fixed based on the last review:

- ✅ Fixed the scroll functionality on some mobile devices.

_John Palacios_
_Microverse Student_